EIPS/eip-1716.md

```diff
@@ -0,0 +1,38 @@
+---
+eip: 1716
+title: "Hardfork Meta: Petersburg"
+author: Afri Schoedon (@5chdn), Marius van der Wijden (https://github.com/MariusVanDerWijden)
```

Comment:
You should just put your GitHub id here if you want to keep the control over the merged EIP.